### PR TITLE
feat: extend design system to support Troms

### PR DESCRIPTION
**Pull request by mortennordseth**
This PR introduces support for a new organization, Troms (represented by the org ID tfk). The changes allow us to build a theme and assets specifically for Troms.

As of now, the colors and illustrations for Troms are identical to those of AtB. This is a temporary measure until Troms creates their own assets.

The choice of tfk as the org ID is in line with the existing naming for NFK. However, if there's a preference for using troms or any other identifier, I'm open to suggestions.

Closes https://github.com/AtB-AS/kundevendt/issues/17074

### CODE REVIEW

**Review by adriansberg — APPROVED** [review]
I think TFK is okay 👍

**Comment by rosvik** [comment]
> The choice of tfk as the org ID is in line with the existing naming for NFK. However, if there's a preference for using troms or any other identifier, I'm open to suggestions.

Maybe TFT? That's used on [wikipedia](https://no.wikipedia.org/wiki/Troms_fylkestrafikk), but can't find any references to it on their website 🤷

**Review by rosvik — APPROVED** [review]

**Comment by mortennordseth** [comment]
> > The choice of tfk as the org ID is in line with the existing naming for NFK. However, if there's a preference for using troms or any other identifier, I'm open to suggestions.
> 
> Maybe TFT? That's used on [wikipedia](https://no.wikipedia.org/wiki/Troms_fylkestrafikk), but can't find any references to it on their website 🤷

Good suggestion, I think TFT is better! The firebase and google cloud projects are named `troms-staging-xyz` though. That might make it confusing 🤔

**Comment by rosvik** [comment]
Hmm, maybe worth asking in #oos-app-web?

**Comment by mortennordseth** [comment]
It was decided to use `troms` as ID: https://mittatb.slack.com/archives/C025NUF4WA0/p1709298925861559

File: packages/theme/src/theme.ts [hunk 1]
```diff
@@ -104,7 +104,7 @@ export enum ThemeVariant {
   AtB,
   Nfk,
   FRAM,
-  TFK,
+  TROMS,
```
Comment (adriansberg) [inline packages/theme/src/theme.ts:107]: Should these be all capitalised? "AtB" and "Nfk" are not.

Comment (mortennordseth) [reply packages/theme/src/theme.ts:107]: I didn't notice that. I can update and use `Troms` instead

**Pull request merged**